src/Walker/Collector.C

```diff
@@ -29,69 +29,13 @@ CkReduction::reducerType PDFMerger;
 using walker::Collector;
 
 void
-Collector::chareOrd( const std::vector< tk::real >& ord )
+Collector::chareOrd( const std::vector< tk::real >& ord,
```

Comment:
I think I would probably make fun of whoever wrote the name sumOrdMomsAndPDF(), but I think it would also save time over having to explain what "chareOrd()" means. 

But I also wouldn't use either function name if I were discussing this. I would give a very high level talk about the algorithm we use and just say "And then we have each collector perform a partial sum of the ordinary moments and PDFs prior to a global reduction".

But I also know that if I am looking through the code, not having to cross reference the manual to understand what chareOrd() does would be nice. Which is really what I am thinking of.